lib/xclarity_client/connection/connection.rb

```diff
@@ -103,7 +107,7 @@ def build(configuration)
         faraday.request        :url_encoded             # form-encode POST params
         faraday.response       :logger, $lxca_log.log   # log requests to STDOUT -- This line, should be uncommented if you wanna inspect the URL Request
         faraday.use            :cookie_jar if configuration.auth_type == 'token'
-        faraday.adapter        Faraday.default_adapter  # make requests with Net::HTTP
+        faraday.adapter        :httpclient  # make requests with HTTPClient
```

Comment:
__Question__
Could you explain why you need to change the adapter here?

Reply:
@douglasgabriel The default Faraday adapter (Net::HTTP) doesn't support some header fields on the response. For instance, we couldn't use `Content-Range` from the response due to this. On the other hand, HTTPClient have pretty much the same features as Net::HTTP and also supports other header fields.